### PR TITLE
Reveal shard count

### PR DESCRIPTION
**Pull request by fppt**
@duckofyork this is the method you need. `graknTx.admin().shardingThreshold()`

### CODE REVIEW

**Comment by duckofyork** [comment]
Thanks! @fppt

File: grakn-kb/src/main/java/ai/grakn/kb/internal/GraknTxAbstract.java [hunk 1]
```diff
@@ -201,6 +202,11 @@ public Keyspace keyspace() {
         return session().keyspace();
     }
 
+    @Override
+    public long shardingThreshold(){
+        return Long.parseLong(session().config().get(GraknConfigKey.SHARDING_THRESHOLD.name()).toString());
```
Comment (aelred) [inline grakn-kb/src/main/java/ai/grakn/kb/internal/GraknTxAbstract.java]: Yeah, we really need to move `GraknEngineConfig` out of `grakn-engine`...

**Comment by fppt** [comment]
@aelred what? The method does not exist yet which is why this PR exists.

**Comment by fppt** [comment]
@duckofyork what needs to change to use this method?

**Comment by duckofyork** [comment]
@aelred  @fppt I'll check out this branch and make the change

File: grakn-kb/src/test/java/ai/grakn/kb/internal/GraknTxTest.java [hunk 2]
```diff
@@ -131,6 +131,11 @@ public void whenGettingSubTypesFromRootMeta_IncludeAllTypes(){
     }
 
     @Test
+    public void whenGettingTheShardingThreshold_TheCorrectValueIsReturned(){
+        assertEquals(100000L, tx.admin().shardingThreshold());
```
Comment (duckofyork) [inline grakn-kb/src/test/java/ai/grakn/kb/internal/GraknTxTest.java]: so we are not using the config in grakn.properties?

Comment (fppt) [reply grakn-kb/src/test/java/ai/grakn/kb/internal/GraknTxTest.java]: Technically this IS the config from grakn.properties. Just in a very roundabout way

**Comment by fppt** [comment]
@aelred re-review please?

**Pull request merged**